tests/foreman/cli/test_discoveredhost.py

```diff
@@ -758,20 +459,6 @@ def test_positive_auto_provision_pxeless_host(self):
         :CaseImportance: Critical
         """
 
-    @pytest.mark.stubbed
-    @pytest.mark.tier3
-    def test_positive_list_discovered_host(self):
-        """List pxe-based and pxe-less discovered hosts
-
-        :id: 3a827080-3fab-4f64-a830-1b41841aa2df
-
-        :expectedresults: Host should be discovered and listed with names.
-
-        :CaseAutomation: NotAutomated
-
-        :CaseImportance: High
-        """
-
     @pytest.mark.stubbed
     @pytest.mark.tier3
     def test_positive_assign_discovery_manager_role(self):
```

Comment:
i'll most probably use non-admin user with discovery roles on all tests after refactoring. I might parameterize the tests to use each role.